samza-core/src/main/java/org/apache/samza/config/BlobStoreConfig.java

```diff
@@ -23,19 +23,56 @@
  * Config related helper methods for BlobStore.
  */
 public class BlobStoreConfig extends MapConfig {
+  
+  private static final String PREFIX = "blob.store.";
+  public static final String BLOB_STORE_MANAGER_FACTORY = PREFIX + "manager.factory";
+  public static final String DEFAULT_BLOB_STORE_MANAGER_FACTORY = "com.linkedin.samza.ambry.AmbryBlobStoreManagerFactory";
+  public static final String BLOB_STORE_ADMIN_FACTORY = PREFIX + "admin.factory";
+  public static final String DEFAULT_BLOB_STORE_ADMIN_FACTORY = "com.linkedin.samza.ambry.AmbryBlobStoreAdminFactory";
+  // Configs related to retry policy of blob stores
+  public static final String BLOB_STORE_RETRY_POLICY_MAX_RETRIES = PREFIX + "retry.policy.max.retries";
```

Comment:
This class is BlobStoreConfig. Can remove "blob store" prefix from variable names and accesors.